packages/react/src/components/builder-content.component.tsx

```diff
@@ -25,6 +24,9 @@ export class BuilderContent<ContentType extends object = any> extends React.Comp
   get builder() {
     return this.props.builder || builder;
   }
+  get name() {
```

Comment:
```suggestion
  get modelName(): string {
```
because explicitness?